### ISSUE TITLE
Develop: Clear notification

### ISSUE THREAD

**Issue by mdconatser** [umgc/fall2022#416]
Depends on #102 (notifications view) and #412 (data objects)

As a user, I would like to be able to clear notifications that I have received when no longer needed. This is just clearing actual notifications of messages, not the subscription itself.

See mockup: https://xd.adobe.com/view/db7cd80e-542d-495d-8909-01d4b2607dbe-8e2f/screen/5327e8da-3192-4805-8c95-d349fad83e70

Acceptance Criteria
--
- User can clear notifications from the Notifications view
- System deletes the `Notification` upon request
- Document any relevant content in the Programmer guide (draft thoughts of usage and importantly the "Why")
- Updates SRS and STP (and other docs if necessary)
- Add/Update relevant unit tests

**Comment by mdconatser** [comment]
tested on android, entered a bug to fix recurring notifications
